### PR TITLE
OrtIssue severity

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/1112)

### CODE REVIEW

File: model/src/main/kotlin/OrtIssue.kt [hunk 1]
```diff
@@ -48,7 +48,12 @@ data class OrtIssue(
...
          */
-        val message: String
+        val message: String,
+
+        /**
+         * The issue's severity.
+         */
+        val severity: Severity = Severity.ERROR
 ) {
     override fun toString() = "${if (timestamp == Instant.EPOCH) "n/a" else timestamp.toString()}: $source - $message"
```
Comment (mnonnenmacher) [inline model/src/main/kotlin/OrtIssue.kt]: The string represenation should also include the severity. Or do you plan to do this in a separate PR because of the refactoring effort involved in changing this?

Comment (sschuberth) [reply model/src/main/kotlin/OrtIssue.kt]: I didn't plan for it at all as I simply forgot about it 😆 About doing it in a separate PR sounds like a good idea to me. Should we do it like that?

Comment (mnonnenmacher) [reply model/src/main/kotlin/OrtIssue.kt]: Yes, I would prefer that.
